promptsource/templates/tydiqa/secondary_task/templates.yaml

```diff
@@ -98,6 +140,19 @@ templates:
 
       {{answers.text | choice}}
 
+      {% elif lang == "indonesian" %}
+
+      Aku lagi cari tau jawaban dari pertanyaan  "{{question}}"
```

Comment:
I found that we use both "Aku" and "Saya". I think we should use only one of them to make it consistent.

Reply:
I can help updating the yaml if you don't mind @afaji.

Reply:
This is by design, since both Aku and Saya is naturally used by native speakers, so I try to cover all possibilities.